.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 jobs:
   build_test:
     docker:
-      - image: debian:stretch
+      - image: debian:latest
```

Comment:
The idea behind using stretch is that we want to support all versions of gcc back to the minimal support that HarfBuzz is supporting. The goal is not to test it with the latest stable version of GCC.

Reply:
@moazin can you please share the reason why you have changed this ?

Reply:
Hi @tjindal, I think the reason why I tried to change it was some dependency not working out to be the right version, I don't recall exactly which one it was and I'm not even sure if this change had fixed it. Enable the GTest part that I had commented out, it's not supposed to be commented, I guess I just did that temporarily for something, and change the version to stretch and see if everything works fine in the CircleCI pipeline. As far I remember, it was a CMake thing, perhaps FetchContent wasn't available or something.

Reply:
changed back to "stretch"

Reply:
changed back to stretch and add the google tests.